layouts/partials/header_home.html

```diff
@@ -1,7 +1,7 @@
 {{ $current := . }}
 {{ $list := .Site.Menus.main | append .Site.Menus.header }}
-  <nav class="fixed lg:absolute top-0 left-0 w-full z-50">
-    <div class="container mx-auto">
+  <nav class="fixed lg:absolute top-0 left-0 w-full z-50 bg-card-light lg:bg-transparent">
```

Comment:
While testing i noticed this weird effect on the homepage mobile menu. This should fix it.